gazel.go

```diff
@@ -88,6 +92,32 @@ func NewVenderor(root, cfgPath string, dryRun bool) (*Venderor, error) {
 
 }
 
+type icacheKey struct {
+	path, srcDir string
+}
+
+type icacheVal struct {
+	pkg *build.Package
+	err error
+}
+
+func (v *Venderor) importPkg(path string, srcDir string) (*build.Package, error) {
+	k := icacheKey{path: path, srcDir: srcDir}
+	v.icacheGuard.RLock()
```

Comment:
what's the need for the guard here? is something secretly using goroutines behind-the-scenes?